organization/views_admin.py

```diff
@@ -1335,24 +1335,63 @@ def organization_edit_process_view(request):
         success = False
 
     # We can use the same url_variables with any processing failures below
-    url_variables = "?issue_analysis_admin_notes=" + str(issue_analysis_admin_notes) + \
-                    "&issue_analysis_done=" + str(issue_analysis_done) + \
-                    "&organization_contact_form_url=" + str(organization_contact_form_url) + \
-                    "&organization_email=" + str(organization_email) + \
-                    "&organization_endorsements_api_url=" + str(organization_endorsements_api_url) + \
-                    "&organization_facebook=" + str(organization_facebook) + \
-                    "&organization_id=" + str(organization_id) + \
-                    "&organization_instagram_handle=" + str(organization_instagram_handle) + \
-                    "&organization_name=" + str(organization_name) + \
-                    "&organization_twitter_handle=" + str(organization_twitter_handle) + \
-                    "&organization_twitter_updates_failing=" + str(organization_twitter_updates_failing) + \
-                    "&organization_type=" + str(organization_type) + \
-                    "&organization_website=" + str(organization_website) + \
-                    "&profile_image_type_currently_active=" + str(profile_image_type_currently_active) + \
-                    "&state_served_code=" + str(state_served_code) + \
-                    "&wikipedia_page_title=" + str(wikipedia_page_title) + \
-                    "&wikipedia_photo_url=" + str(wikipedia_photo_url) + \
-                    "&google_civic_election_id=" + str(google_civic_election_id)
+
+    url_variables = "?n=1"
+    
+    if issue_analysis_admin_notes is not False:
+        url_variables += "&issue_analysis_admin_notes"
```

Comment:
Hi @hesamshayegan you forgot to add on the `+ str(issue_analysis_admin_notes)` part.

Reply:
Thank, @DaleMcGrew! Just fixed them.